lib/rubocop/cop/bundler/gem_filename.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Bundler
+      # This cop verifies that a project contains Gemfile or gems.rb file and correct
+      # associated lock file based on the configuration.
+      #
+      # @example RequiresGemfile: true (default)
```

Comment:
I'd suggest "EnforcedStyle" with the options "Gemfile" and "gems.rb". To me at least, that's much easier to understand.

Reply:
Absolutely agree, and it's more consistent with other cops that way.